streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBTimestampedStoreTest.java

```diff
@@ -139,70 +212,70 @@ public void shouldMigrateDataFromDefaultToTimestampColumnFamily() throws Excepti
 
     private void iteratorsShouldNotMigrateData() {
         // iterating should not migrate any data, but return all key over both CF (plus surrogate timestamps for old CF)
-        final KeyValueIterator<Bytes, byte[]> itAll = rocksDBStore.all();
```

Comment:
Added some exception handling improvements to close all resources properly in case of failure.